test-snapshots/spago.yaml

```diff
@@ -15,53 +15,4 @@ workspace:
     args:
       - build
   package_set:
-    registry: 19.1.0
-  extra_packages:
-    prelude:
-      git: https://github.com/purescm/purescript-prelude.git
-      ref: b250d70c7bc6528c12e0598c50ad87b7267ccfa9
-      dependencies: []
-    partial:
-      git: https://github.com/purescm/purescript-partial.git
-      ref: d8b6daf068e4aab0ad85a42d0f890ed37f416923
-      dependencies: []
-    functions:
-      git: https://github.com/purescm/purescript-functions.git
-      ref: 94e0a9a3b7f831abebcb1bf65587fca03f04e034
-      dependencies:
-        - prelude
-    effect:
-      git: https://github.com/purescm/purescript-effect.git
-      ref: 8aee2c2fbf5425bf8fa4537a5ad2070170622acf
-      dependencies:
-        - prelude
-    console:
-      git: https://github.com/purescm/purescript-console.git
-      ref: 4a30427e490b6f38be16bb5e0bc8f68ecc1c9999
-      dependencies:
-        - effect
-        - prelude
-    assert:
-      git: https://github.com/purescm/purescript-assert.git
-      ref: 348b91f4542cf75efab232e8eb52dd5d3e6a9435
-      dependencies:
-        - console
-        - effect
-        - prelude
-    refs:
-      git: https://github.com/purescm/purescript-refs.git
-      ref: 2ac82d3867e8ac02df03b6a04d635e0acd4b95db
-      dependencies:
-        - effect
-        - prelude
-    record:
-      git: https://github.com/purescm/purescript-record.git
-      ref: 8a993da0d328f72167a4742e0e2a428a790510bc
-      dependencies:
-        - functions
-        - prelude
-        - unsafe-coerce
-    unsafe-coerce:
-      git: https://github.com/purescm/purescript-unsafe-coerce.git
-      ref: 20dbd7797e985b631459b8dbd4072cf3096ee1da
-      dependencies: []
+    url: https://raw.githubusercontent.com/purescm/purescm/package-set/package-sets/1.0.0.json
```

Comment:
I was thinking I could just point this to `master` before merging.

Reply:
Oh but then it won't build :thinking:

Reply:
Pointing to master is a questionable choice in any case - sure, we keep a hash of the package set, but it would make for a somewhat unpredictable upgrade. We should use a commit hash rather than a branch

Reply:
I cannot point to a commit-ref that exists in `master`, but now just pointing to a commit in this branch.

Reply:
Yeah, that's good - it should stay available even after we delete the branch

Reply:
Actually this probably won't work. I just need to make another PR once this one is merged to point to a proper ref.

Reply:
This is all a little dull actually - there's no reason why one should not be allowed to point to a filesystem path 😄

Reply:
Yeah, I was looking for this in the spago docs :)

Reply:
I opened https://github.com/purescript/spago/issues/1031